ddtrace/appsec/processor.py

```diff
@@ -11,7 +11,7 @@
 from ddtrace.ext import SpanTypes
 from ddtrace.internal.logger import get_logger
 from ddtrace.internal.processor import SpanProcessor
-from ddtrace.utils.formats import get_env
+from ddtrace.internal.utils.formats import get_env
```

Comment:
need a rebase from `master` on this one?

Reply:
This change might conflict with Zach's env deprecations

Reply:
This change was also now made on master. Keeping it here but will need to be dropped at some point as we merge #2996.